tsconfig.base.json

```diff
@@ -8,6 +8,8 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "importHelpers": true,
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
```

Comment:
What does this setting do?

Reply:
So, @bit172 had something like `import * as cookieParser from 'cookieParser'` because it isn't an export default , so we can import it as `import cookieParser from 'cookieParser'` with this.